packages/react/src/components/List/List.mdx

```diff
@@ -29,29 +29,93 @@ import { List } from '@digdir/designsystemet-react';
 </List.Root>;
 ```
 
-## Eksempler på bruk av `List`
+## Varianter av `List`
 
-### Uten heading
+<br />
+
+### Unummerert liste
```

Comment:
```suggestion
### Usortert liste
```
Dette burde være "usortert liste", sia det er det me kaller det over

Reply:
tenker vi går for nummerert og unummerert konsekvent

Reply:
Det bryter med korleis det semantiske elementet er i kode. Der bruker du `ul` og `ol`, som står for `unordered list` og `ordered list`

Reply:
ok, da går vi får sortert og usortert.

Reply:
Fra mdn:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ol

Reply:
Som du kan sjå på `ol` elementet, så kan du endre kva lista er sortert med - slik at det ikkje alltid er nummer.
![image](https://github.com/digdir/designsystemet/assets/14057205/bd7cf08a-7319-4fc5-a179-4bb23a7cf2fb)